apps/zotonic_mod_development/src/mod_development.erl

```diff
@@ -96,6 +98,48 @@ observe_request_context(#request_context{ phase = refresh }, Context, _Context)
 observe_request_context(#request_context{ phase = _ }, Context, _Context) ->
     Context.
 
+%% @doc Runs Chrome opening it in the site URL.
+%% Ignore certificate errors and defines the site as secure, helpful to run Web Workers.
+%% For extra args @see https://peter.sh/experiments/chromium-command-line-switches/
+%% Common args:
+%%   --incognito            Launches Chrome directly in Incognito private browsing mode
+%%   --purge-memory-button  Add purge memory button to Chrome
+%%   --multi-profiles       Enable multiple profiles in Chrome
+%% e.g.
+%% ``` mod_development:chrome(foo, ["--incognito", "--start-maximized"]). '''
+-spec chrome(SiteOrContext :: atom() | z:context()) -> {ok, port()} | {error, term()}.
+chrome(SiteOrContext) ->
+    chrome(SiteOrContext, []).
+
+-spec chrome(SiteOrContext :: atom() | z:context(), ExtraArgs :: [string()]) -> {ok, port()} | {error, term()}.
+chrome(#context{} = Context, ExtraArgs) ->
+    case os:find_executable("google-chrome") of
```

Comment:
I suppose need also add `os:find_executable("chromium-browser")` for Linux.

Reply:
I think we can try to pick the executable in this order `["chromium-browser", "google-chrome"]`.